wandb/cli/cli.py

```diff
@@ -1571,3 +1572,37 @@ def gc(args):
     click.echo(
         "`wandb gc` command has been removed. Use `wandb sync --clean` to clean up synced runs."
     )
+
+
+@cli.command(context_settings=CONTEXT, help="Verify your local instance")
+@click.option("--host", default=None, help="Test a specific instance of W&B")
+def verify(host):
+    os.environ["WANDB_SILENT"] = "true"
+    api = _get_cling_api()
+
+    if not os.path.exists("./wandb/verify_test"):
+        os.makedirs("./wandb/verify_test")
```

Comment:
Hmmm, since we're changing into a temp directory why are we doing this?

Reply:
This can be removed